tests/test_multiple_date.py

```diff
@@ -300,6 +319,47 @@ def elem(timestamp, value):
 
         return TiledRasterLayer.from_numpy_rdd(LayerType.SPACETIME, rdd, metadata)
 
+    def get_test_layer(self):
```

Comment:
note that there's also some test fixtures somewhere, perhaps those could be improved & reused, or this method could be converted into such a fixture?

Reply:
True, moved!